dex-auth/dex-crds/base/params.env

```diff
@@ -2,8 +2,8 @@
 dex_domain=dex.example.com
 # Set issuer to https if tls is enabled
 issuer=http://dex.example.com:32000
-static_email=admin@example.com
```

Comment:
This is a default change. User who not explicitly specify credentials may need it. Please don't touch it.

Reply:
This is used by a few  kfdefs

Reply:
Yeah I agree, fixed